Packs/Edgescan/Integrations/Edgescan/README.md

```diff
@@ -0,0 +1,9183 @@
+                "layer": "network",
+                "location": "192.168.0.1",
+                "location_specifier_id": 191,
+                "name": "SSL Version 2 (v2) Protocol Detection",
+                "pci_compliance_status": "fail",
+                "risk": 3,
+                "severity": 4,
+                "status": "closed",
+                "threat": 3,
+                "updated_at": "2021-04-02T15:35:26.107Z"
+            },
+            {
+                "altered_score": false,
+                "asset_id": 164,
+                "asset_name": "Edgescan Internal Server Farm",
+                "confidence": null,
+                "created_at": "2021-04-02T15:35:26.140Z",
+                "cves": [
+                    "CVE-2015-0204"
+                ],
+                "cvss_score": 4.3,
+                "cvss_v2_score": null,
+                "cvss_v2_vector": null,
+                "cvss_vector": "AV:N/AC:M/Au:N/C:N/I:P/A:N",
+                "cvss_version": 2,
+                "date_closed": null,
+                "date_opened": "2019-08-15T10:20:51.058Z",
+                "definition_id": 137,
+                "id": 52493,
+                "label": null,
+                "last_pci_exception": null,
+                "layer": "network",
+                "location": "192.168.0.1",
+                "location_specifier_id": 191,
+                "name": "SSL/TLS EXPORT_RSA <= 512-bit Cipher Suites Supported (FREAK)",
+                "pci_compliance_status": "fail",
+                "risk": 3,
+                "severity": 3,
+                "status": "open",
+                "threat": 3,
+                "updated_at": "2021-06-11T08:37:55.020Z"
+            },
+            {
+                "altered_score": false,
+                "asset_id": 164,
+                "asset_name": "Edgescan Internal Server Farm",
+                "confidence": null,
+                "created_at": "2021-04-02T15:35:26.206Z",
+                "cves": [
+                    "CVE-1999-0024"
+                ],
+                "cvss_score": 5,
+                "cvss_v2_score": null,
+                "cvss_v2_vector": null,
+                "cvss_vector": "AV:N/AC:L/Au:N/C:N/I:P/A:N",
+                "cvss_version": 2,
+                "date_closed": null,
+                "date_opened": "2019-08-15T10:20:51.058Z",
+                "definition_id": 47,
+                "id": 52494,
+                "label": null,
+                "last_pci_exception": null,
+                "layer": "network",
+                "location": "10.0.0.2",
+                "location_specifier_id": 192,
+                "name": "DNS Server Recursive Query Cache Poisoning Weakness",
+                "pci_compliance_status": "fail",
+                "risk": 3,
+                "severity": 3,
+                "status": "open",
+                "threat": 3,
+                "updated_at": "2021-06-11T08:37:55.035Z"
+            },
+            {
```

Comment:
Here as well, let's keep only two examples

Reply:
Not sure, I think this is okay now?

Reply:
The outputs here is ok but too long. Let's remove most of it and keep only two examples

Reply:
I did what I could